lib/config/environments.js

```diff
@@ -69,14 +67,16 @@ module.exports = {
                 this.define(`${pluginName}/${envName}`, plugin.environments[envName]);
             });
         }
-    },
+    }
 
     /**
      * Resets all environments. Only use for tests!
      * @returns {void}
      */
     testReset() {
```

Comment:
Is `testReset` needed anymore? It seems like it would be better to create a new instance instead.